packages/material-design-icons/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@automattic/material-design-icons",
+	"version": "1.0.0",
+	"description": "Material icons in SVG format.",
+	"main": "src/index.js",
+	"module": "src/index.js",
+	"sideEffects": false,
+	"keywords": [
+		"wordpress"
+	],
+	"author": "Brad Griffith <brad.griffith@automattic.com>",
+	"contributors": [],
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "Apache-2.0",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/material-design-icons"
+	},
+	"publishConfig": {
+		"access": "private"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"src"
+	],
+	"devDependencies": {
+		"grunt": "^0.4.5",
+		"grunt-cli": "^1.2.0",
+		"grunt-svgstore": "^1.0.0"
+	},
+	"scripts": {
+		"build": "grunt --verbose"
```

Comment:
could we do this with the [svgstore cli](https://github.com/svgstore/svgstore-cli) and not pull in grunt?

Reply:
Thank you.  Did not stumble across this and just followed the gridicons example.  Very happy to not have to touch grunt.

Reply:
Made the switch in 426e7a8.